src/relay/transforms/fuse_ops.cc

```diff
@@ -809,8 +811,31 @@ std::vector<GraphPartitioner::Group*> GraphPartitioner::Partition(
 
 class FuseMutator : private MixedModeMutator {
  public:
+  int GetFuseOptLevel() { return fuse_opt_level_; }
+  size_t GetMaxFuseDepth() { return max_fuse_depth_; }
+  bool GetLinkParams() { return link_params_; }
+
+  FuseMutator* SetFuseOptLevel(int fuse_opt_level) {
```

Comment:
Why cant we just use a constructor with defaults to set these ?

Reply:
Amended.